tools/generate_bech32_tests.py

```
@@ -0,0 +1,224 @@
+#!/usr/bin/env python3
+"""
+Single comprehensive tool to generate bech32 test cases for ledger-zxlib.
+Uses reference implementation from sipa/bech32 to generate test vectors
+that can be verified with online tools.
+"""
+
+import random
```

Ruff (0.12.2)

`random` imported but unused

Remove unused import: `random`

(F401)